doc/advanced/experimental.md

```diff
@@ -30,6 +30,7 @@ Both kinds of experimental features can be enabled simultaneously.
 | ONEDNN_EXPERIMENTAL_UKERNEL                | Enable experimental microkernel APIs and functionalities.          |
 | ONEDNN_EXPERIMENTAL_PROFILING              | Enable experimental profiling API.                                 |
 | ONEDNN_EXPERIMENTAL_LOGGING                | Enable experimental logging support for oneDNN verbose mode.       |
+| DNNL_EXPERIMENTAL_SELF_SUFFICIENT_L0       | Enable experimental functionality in SYCL to allow the removal of the OpenCL dependency for the L0 backend.       |
```

Comment:
`ONEDNN_` for consistency? (might require updating cmake/dnnl_compat.cmake as well)